spec/unit/concerns/composite_api_spec.rb

```diff
@@ -139,5 +139,31 @@
       double('Faraday::Response', body: { 'compositeResponse' => [] })
     end
     it_behaves_like 'composite requests'
+
+    it 'has response in CompositeAPIError' do
```

Comment:
Should this go with the shared examples? This case can happen whenever `composite` is called with `allOrNone: true` - not exclusive to `composite!`.

Reply:
you are right. forgot to update this.